extra-dev/packages/coq-mathcomp-analysis/coq-mathcomp-analysis.dev/opam

```diff
@@ -1,41 +1,45 @@
 opam-version: "2.0"
-maintainer: "pierre-yves@strub.nu"
+maintainer: "Reynald Affeldt <reynald.affeldt@aist.go.jp>"
+version: "dev"
+
 homepage: "https://github.com/math-comp/analysis"
-bug-reports: "https://github.com/math-comp/analysis/issues"
 dev-repo: "git+https://github.com/math-comp/analysis.git"
+bug-reports: "https://github.com/math-comp/analysis/issues"
 license: "CECILL-C"
+
+synopsis: "An analysis library for mathematical components"
+description: """
+This repository contains an experimental library for real analysis for
+the Coq proof-assistant and using the Mathematical Components library."""
+
+build: [make "-j%{jobs}%"]
+install: [make "install"]
+depends: [
+  "coq" { (>= "8.13" & < "8.15~") | (= "dev") }
+  "coq-mathcomp-ssreflect" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-fingroup" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-algebra" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-solvable" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-field" { (>= "1.12.0" & < "1.14~") | (= "dev") }
+  "coq-mathcomp-finmap" { (>= "1.5.1" & < "1.6~") | (= "dev") }
+  "coq-mathcomp-bigenough" { (>= "1.0.0") }
+  "coq-hierarchy-builder" { (>= "0.10.0") }
```

Comment:
```suggestion
  "coq" { (>= "8.13") }
  "coq-mathcomp-ssreflect" { (>= "1.12.0") }
  "coq-mathcomp-fingroup" { (>= "1.12.0") }
  "coq-mathcomp-algebra" { (>= "1.12.0") }
  "coq-mathcomp-solvable" { (>= "1.12.0") }
  "coq-mathcomp-field" { (>= "1.12.0") }
  "coq-mathcomp-finmap" { (>= "1.5.1") }
  "coq-mathcomp-bigenough" { (>= "1.0.0") }
  "coq-hierarchy-builder" { (>= "0.10.0") }
```
How about only going with lower bounds here in `extra-dev` @affeldt-aist @proux01? Then you shouldn't have to update the package as often.

Reply:
Very good point! (although for mathcomp, we can hope next release will be 2.0 so we'll have to update the lower bound anyway ;-) )